test/AdoNetCore.AseClient.Tests/Integration/Connection/TlsConnectionTests.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.IO;
+using Dapper;
+using NUnit.Framework;
+
+namespace AdoNetCore.AseClient.Tests.Integration.Connection
+{
+    [TestFixture]
```

Comment:
I'd probably mark this fixture as `Explicit = true`

Reply:
The test is explicit in a sense - it only runs if the test suite is configured for with a TLS endpoint.